src/modules/translation/languages/en.json

```diff
@@ -491,6 +492,21 @@
     "total_land": "Total Land",
     "empty_land": "Empty Land"
   },
+  "land_ens_page" : {
+    "title": "Set Name",
+    "subtitle": "Set the ethereum name service to your land",
```

Comment:
I think the `ethereum name service` part is not very user friendly, maybe something more literal like: `Assign one of your Names to act as a link to your land.` ?

Reply:
Done! thanks!